mesonbuild/interpreter.py

```diff
@@ -874,7 +875,28 @@ def get_define_method(self, args, kwargs):
         extra_args = self.determine_args(kwargs)
         deps = self.determine_dependencies(kwargs)
         value = self.compiler.get_define(element, prefix, self.environment, extra_args, deps)
-        mlog.log('Checking for value of define "%s": %s' % (element, value))
+        mlog.log('Fetching value of define "%s": %s' % (element, value))
+        return value
+
+    def get_return_value_method(self, args, kwargs):
```

Comment:
This is a poor name and very confusing. "Return value" already has a specific and unambiguous meaning: the integer that the executable returns from main.